style.css

```diff
@@ -15,19 +15,23 @@ header {
 
 .logo {
   text-decoration: none;
-  font-family: Arial, Helvetica, sans-serif;
+  font-family: sans-serif;
   font-size: 30px;
   font-weight: bolder;
   color: #28352f;
 }
 
 .background-image {
   background-image: url("asset/images/header-illsutration-mobile@2x.png");
-  min-height: 100vh;
   background-size: cover;
+  min-height: 100vh;
   max-width: 100%;
 }
 
+.desktop-header {
+  display: none;
+}
```

Comment:
- [ ] Please add the correct Height and margin-top value for your header in the mobile view and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/165749667-8ada4899-cf4b-4d67-9bf1-04b0b872f933.png)  | ![image](https://user-images.githubusercontent.com/68796232/165749617-4643c4de-7558-42ab-ab84-b3d2fc8dd1f9.png) |